lib/Models/ImageryLayerCatalogItem.js

```diff
@@ -547,8 +547,11 @@ function onClockTick(catalogItem, clock) {
         fixNextLayerOrder(catalogItem);
         ImageryLayerCatalogItem.disableLayer(catalogItem, catalogItem._imageryLayer);
 
-        catalogItem._imageryLayer = catalogItem._nextLayer;
-        catalogItem._imageryLayer.imageryProvider.enablePickFeatures = true;
+        if (defined(catalogItem._nextLayer))
+        {
+            catalogItem._imageryLayer = catalogItem._nextLayer;
```

Comment:
I think this line should be outside the `if`.  We don't want the previous layer lingering around if there's no next layer.